dist/Makefile

```diff
@@ -0,0 +1,11 @@
+VERSION=1.8.0
```

Comment:
This should be a `Makefile.am` and this can be used to get `VERSION` without hard coding it.